brave-people.json

```diff
@@ -513,5 +513,10 @@
   "y" : 12,
   "color" : "#3489fd",
   "username" : "Tsanfer"
+}, {
+  "x" : 0,
+  "y" : 13,
+  "color" : "#00CCFF",
+  "username" : "queel"
```

Comment:
```suggestion
  "username" : "wwx22666"
```

Reply:
thanks，I realize that I should use github username